utils/ssh.py

```diff
@@ -119,10 +122,6 @@ def connect(self, hostname=None, **kwargs):
             # Only install ssh keys if they aren't installed (or currently being installed)
             return super(SSHClient, self).connect(**self._connect_kwargs)
 
-    def open_sftp(self, *args, **kwargs):
```

Comment:
Yeah, this should not get removed. I added that to `.connect()` inside. Doing this will nuke some tests I believe.